src/main/resources/application.properties

```diff
@@ -0,0 +1,6 @@
+server.port = 8081
+spring.datasource.url=jdbc:postgresql://localhost:5432/postgres
+spring.datasource.username= postgres
+spring.datasource.password= admin
```

Comment:
I don't usually ask for this in trainings. But let's use enviroment variables for this properties